gbpservice/nfp/orchestrator/drivers/orchestration_driver.py

```diff
@@ -29,6 +29,8 @@
 
 from gbpservice.nfp.core import log as nfp_logging
 
+from gbpservice.nfp.core import executor as nfp_executor
+
```

Comment:
Is it ok, to call framework API in driver, generally module interacts with framework.
Can we avoid this.